src/app/Main/Body/QueryPage/Services/ContainerServices/fetchInitialData.service.tsx

```diff
@@ -42,7 +42,9 @@ export async function fetchInitialDataService(): Promise<{
         const adaptedSubFilters: ClientSingleFilterConfig[] = [];
         adaptedFilterProp.filters.forEach((adaptedFilter) => {
             if (adaptedFilter.parent !== undefined) {
-                adaptedSubFilters.push(adaptedFilter);
+                if (adaptedFilter.values.length > 1) {
```

Comment:
What is the meaning of sub-filters with only one value returned by the API?
In general it is good to check the data from the API before feeding it into your app.